parameter_files/fates_params_default.cdl

```diff
@@ -1098,12 +1220,16 @@ data:
 
  fates_cohort_fusion_tol = 0.08 ;
 
+ fates_cohort_age_fusion_tol = - ;
```

Comment:
@rgknox @JessicaNeedham  can this parameter also serve as a switch to turn off cohort age tracking entirely or do we need to add a separate variable to do that?

Reply:
Yes this parameter could work as the switch. It doesn't have that function yet but I can add it in.